data/json/mutations/mutations.json

```diff
@@ -922,7 +922,7 @@
     "id": "SLOWHEALER3",
     "name": { "str": "Imperceptive Healer" },
```

Comment:
This should really be "Imperceptible Healer" - would this change cause a problem? (And all of the "[Foo] Healer" ones, but especially this one, would probably sound better as "[Foo] Healing".)

Reply:
I'm not entirely sure. My gut says no; the rest of the code seems to care more about the ID than the friendly name. The only other functional code that uses the string is a test file; my gut says that can be disregarded. @anothersimulacrum -- your thoughts?